sql/parser/eval.go

```diff
@@ -448,6 +448,8 @@ type EvalContext struct {
 	NodeID        uint32
 	StmtTimestamp time.Time
 	TxnTimestamp  time.Time
+	Location      string
```

Comment:
There is no requirement that the members of `EvalContext` need to be primitive types or structs. We could very easily add a `GetTimeZone` function pointer.